examples/example_sumArray.cpp

```diff
@@ -137,7 +137,7 @@ int main() {
 
     // 2B. Set kernel string, header files and compile options
     Headers headers;
-    headers.insert(Header{"cuda_runtime.h"});
+    // headers.insert(Header{"cuda_runtime.h"});
```

Comment:
I'm actually really unsure why this was here to begin with?
But I also can't find this file automatically under windows like I can under linux, so I just removed it.